libraries/app/include/graphene/app/database_api.hpp

```diff
@@ -444,6 +444,14 @@ class database_api
        */
       vector<call_order_object> get_call_orders(const std::string& a, uint32_t limit)const;
 
+      /**
+       * @brief Get call orders from a given account
+       * @param account_name_or_id Account name or ID to get objects from
+       * @param limit Maximum number of objects to retrieve
+       * @return The call orders of the account
+       */
+      vector<call_order_object> get_call_orders_by_account(const std::string& account_name_or_id, uint32_t limit)const;
```

Comment:
Need a `start` or `lower_bound` parameter to be able to fetch next pages.

Reply:
done at https://github.com/bitshares/bitshares-core/pull/1749/commits/38c3cef72d6379b50b2268fe908a84d467ca6b93